plugin/pkg/managedseed/validator/admission.go

```diff
@@ -332,6 +332,19 @@ func (v *ManagedSeed) admitSeedSpec(spec *gardencore.SeedSpec, shoot *gardencore
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("provider", "region"), spec.Provider.Region, fmt.Sprintf("seed provider region must be equal to shoot region %s", shoot.Spec.Region)))
 	}
 
+	// Initialize and validate settings
+	shootVPAEnabled := gardencorehelper.ShootWantsVerticalPodAutoscaler(shoot)
+	if spec.Settings == nil || spec.Settings.VerticalPodAutoscaler == nil {
+		if spec.Settings == nil {
+			spec.Settings = &gardencore.SeedSettings{}
+		}
```

Comment:
Can you move it up before the first `if`?

Reply:
I probably could, but should I? I find the code as it's written easier to understand - we only change something in spec if we are sure we want to apply a default, not otherwise.
I am anxious to get this reviewed and merged to fix the broken dev - could we focus on the significant parts please?